styles/helpers/tooltip.css

```diff
@@ -0,0 +1,137 @@
+button.tooltip {
+  overflow: visible;
+}
+.tooltip {
+  @mixin tooltip;
```

Comment:
we should have the mixin handle all the css.  We need to think of mixins as standalone functions that can work by themselves and then the helper classes just use the mixins